Fabric.Authorization.AccessControl/src/app/modules/access-control/custom-group/custom-group.component.html

```diff
@@ -113,8 +113,16 @@ <h3 class="h3">
             </tr>
         </tbody>
     </table>
+
+    <hc-progress-spinner [hidden]="(!this.customGroups || this.missingManageAuthorizationPermission) ? 'hidden' : 'visible' "
+      [diameter]="25"
+      [color]="blue"
+      [hasChannel]="true"
+      [isCentered]="false"
+      [isDeterminate]="false">
+    </hc-progress-spinner>
     <div class="buttons">
-        <button hc-button color="primary" (click)="save()" [disabled]="missingManageAuthorizationPermission" [title]="disabledSaveReason">Save</button>
+        <button hc-button color="primary" (click)="save()" [disabled]="!this.customGroups || this.missingManageAuthorizationPermission" [title]="disabledSaveReason">Save</button>
```

Comment:
When the page first loads, the save button is enabled.  Need it enabled later, so I added this.